SKA-ScienceDataProcessor/integration-prototype
# Adds a health check command when deploying the Tango SQL container

## Description:
Tidy update, which adds a Docker health check command when deploying the Tango Database SQL container as part of the SIP demo deployment.

## Testing instructions:
Start the docker demo stack with:

```bash
docker stack deploy -c deploy/demos_23_11_18/docker-compose.yml sip
```

After a short while (20-30s) the container should report as healthy. This can be checked with the following command:

```bash
docker ps -f name=sip_tc_tango_mysql
```
which should report a `status` of healthy.
